### DISCUSSION TITLE
Easy term on integrated terminal in vscode has a weird UI bug

### DISCUSSION THREAD

**Discussion by KenGondor** [JanDeDobbeleer/oh-my-posh#4001] [Troubleshoot]
Not sure if this is due to my laptop running amd, but my laptop running on intel does not have this issue.

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/47976054/f676f8f8-3b08-438e-859d-0490a612c421)

The prompt has gap and it gets chopped in half (the path information disappears) when i run `clear`. This doesn't happen on the windows terminal application. Any ideas what could cause this and if I can resolve it in any way? Thanks alot!

**Comment by JanDeDobbeleer** [comment]
Line height settings? Definitely a terminal setting issue.

**Reply by KenGondor** [reply]
I'll  thinker around and give an update. The line settings doesn't change anything for me.
